jenkins/bootstrap.py

```diff
@@ -313,8 +321,11 @@ def finish(gsutil, paths, success, artifacts, build, version, repo):
         'timestamp': time.time(),
         'result': 'SUCCESS' if success else 'FAILURE',
         'passed': bool(success),
-        'metadata': metadata(repo, artifacts),
+        'metadata': meta,
     }
+    if version:
+        data['job-version'] = version
```

Comment:
Once gubernator and testgrid decide on where to find this value let's delete all the redundant places it exists (the ambiguously named `version` first and foremost).

Good point though that it might just be easiest/best to make testgrid/gubernator find it in finished.json['metadata']['job-version']